simple/README.md

```diff
@@ -0,0 +1,26 @@
+# Simplified setup for WSO2 kubernetes API Manager
+
+![WSO2 API Manager Deployment](extra/apim_simple.png)
+
+## Contents
+* Prerequisites
+
+* Quick Start Guide
+
+## Prerequisites
+* Install [Kubernetes  Client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the steps provided in the following quick start guide.
+* An already setup Kubernetes cluster. If you are unfamiliar with this context, you can use [this guide](https://kubernetes.io/docs/setup/pick-right-solution/) to set up the cluster.
+* WSO2 subscribed users can run **wso2am.sh** with latest updates by providing their subscription username and password. If you do not possess an active WSO2 subscription already, run **wso2am_open.sh** which does not require subscription credentials. 
+
+## Quick Start Guide
+1. Download simplified kubernetes setup for WSO2 API Manager (either **wso2am.sh** or **wso2am_open.sh**).From this point forward the steps are being described for wso2am.sh. If you have downloaded wso2am_open.sh please substitute wso2am.sh with wso2am_open.sh for every command.  
+
+1. In the command line, move into the directory where you have downloaded the simplified kubernetes-apim setup. (Usually, the file would be in the “Downloads” directory unless you have changed the default directory to somewhere else.)
```

Comment:
Why do all the steps are having the `1.`?, shall we change it as appropriate.

Reply:
<img width="1010" alt="Screen Shot 2019-04-29 at 2 23 10 PM" src="https://user-images.githubusercontent.com/47025945/56885568-612ed100-6a8a-11e9-9c21-865d9dc2b87a.png">
this is how it looks at the end.